### DISCUSSION TITLE
How to show the user the benefits of logging in

### DISCUSSION THREAD

**Discussion by Meesch** [UUDigitalHumanitieslab/I-analyzer#1613] [General]
Currently, there are some corpora that are only accessible to registered users from the UU, and in the future there might be more features only accesible to registered users (I'm thinking of large downloads, adding your own corpora etc). What is the best way top communicate this to the user?

**Comment by jgonggrijp** [comment]
Suggestion: place a question mark icon next to the register button. When hovered, display a tooltip that describes the benefits in one sentence.
